packages/core/src/components/Search/__stories__/Search.mdx

```diff
@@ -53,6 +69,53 @@ There are three sizes available: Small (32px), Medium (40px), and Large (48px).
 
 <Canvas of={SearchStories.Sizes} />
 
+### With additional action
+
+The search field can contain one more search-related action (like filtering). This action should be represented by an icon button on the right.
+
+<Canvas of={SearchStories.WithAdditionalAction} />
+
+### With clear button
+
+Once the user starts typing, include a clear button that lets the user delete their current search quickly.
+
+<Canvas of={SearchStories.WithClearButton} />
```

Comment:
Is it an actual variant or the regular behavior?

Reply:
it is regular, but design insisted

Reply:
But it's really confusing, having the word "include" means it's opt-in
I don't like it at all